dockstore-integration-testing/src/test/java/io/dockstore/webservice/SearchResourceIT.java

```diff
@@ -35,7 +37,7 @@
 import org.junit.experimental.categories.Category;
 import org.junit.rules.ExpectedException;
 
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
I think there is a rule in the IDE for not adding wildcard imports, I don't think this is in the primer but probably needs to be.

Reply:
It might be implicit in https://github.com/ga4gh/dockstore/blob/develop/codestyle.xml 
I don't see an explicit tag for it, but this has been our convention due to https://rules.sonarsource.com/java/RSPEC-2208 kind of thing